reactor-core/src/withMicrometerTest/java/reactor/core/publisher/MonoMetricsTest.java

```diff
@@ -373,48 +363,20 @@ public void noRequestTrackingEvenForNamedSequence() {
 		Mono<Integer> source = Mono.just(10)
 		                           .name("foo")
 		                           .hide();
-		new MonoMetrics<>(source, registry)
-				.block();
 
-		DistributionSummary meter = registry.find(METER_REQUESTED)
+		new MonoMetrics<>(source, registry).block();
+
+		DistributionSummary meter = registry.find("foo" + METER_REQUESTED)
 		                                    .summary();
 
 		assertThat(meter).as("global find").isNull();
 
-		meter = registry.find(METER_REQUESTED)
-		                .tag(TAG_SEQUENCE_NAME, "foo")
+		meter = registry.find("foo" + METER_REQUESTED)
 		                .summary();
 
 		assertThat(meter).as("tagged find").isNull();
 	}
 
-	//see https://github.com/reactor/reactor-core/issues/1425
```

Comment:
I'd still try to keep both of these tests, validating that a `Mono.just` and a `Mono.error` both have the same default set of tags. Just remove the `.name` (or make it the same). Same in the flux test.